build_scripts/build_usd.py

```diff
@@ -386,7 +392,7 @@ def RunCMake(context, force, extraArgs = None):
     if generator is not None:
         generator = '-G "{gen}"'.format(gen=generator)
 
-    if IsVisualStudio2019OrGreater():
+    if IsVisualStudio2022OrGreater() or IsVisualStudio2019OrGreater():
```

Comment:
Since you are making a change here I have a fix for a problem I found where on Windows you use a generate other than Visual Studio (such as Ninja).
`if generator and 'Visual Studio' in generator and IsVisualStudio2019OrGreater():`

Also is the or really needed? The function is IsVisualStudio2019**OrGreater**() so shouldn't it apply to VS2022 already?

Reply:
Yes, I think you are right, this is not needed